inference-engine/include/cpp/ie_cnn_network.h

```diff
@@ -187,6 +187,16 @@ class INFERENCE_ENGINE_API_CLASS(CNNNetwork) {
         return actual->getFunction();
     }
 
+    /**
+     * @brief Returns constant nGraph function
+     *
+     * @return constant nGraph function
+     */
+    void setWeights(const Blob::CPtr& weights) {
```

Comment:
Why do we need to extend the public API?
At the current moment we read model and weights at the same time.

Reply:
Right, this one should be private or protected as not intended to be used thru public APIs. I'll fix that.

Reply:
Fixed in the latest commit.